names/data/names_data.hcl

```diff
@@ -0,0 +1,16131 @@
+# Copyright (c) HashiCorp, Inc.
+
+service "accessanalyzer" {
+
+  sdk {
+    id             = "AccessAnalyzer"
+    client_version = [2]
+  }
+
+  names {
+    aliases             = [""]
+    provider_name_upper = "AccessAnalyzer"
+    human_friendly      = "IAM Access Analyzer"
+  }
+
+  client {
+    go_v1_client_typename = "AccessAnalyzer"
+    skip_client_generate  = false
+  }
+
+  env_var {
+    deprecated_env_var = ""
+    tf_aws_env_var     = ""
+  }
```

Comment:
@ThomasZalewski small detail: since these are optional, is it possible to exclude them from the config and have the parser still work? Same thought with all optional fields such as empty lists